Tests/scripts/validate_files_structure.py

```diff
@@ -351,47 +324,174 @@ def changed_context(file_path):
     return False
 
 
+def playbook_valid_in_id_set(file_path, playbook_set):
+    playbook_data = get_playbook_data(file_path)
+    file_id = playbook_data.keys()[0]
+
+    is_found = False
+    for playbook in playbook_set:
+        playbook_id = playbook.keys()[0]
+        if playbook_id == file_id:
+            is_found = True
+            if playbook != playbook_data:
+                print_error("You have failed to update id_set.json with the data of {} "
+                            "please run `python Tests/scripts/update_id_set.py`".format(file_path))
+                return False
+
+    if not is_found:
+        print_error("You have failed to update id_set.json with the data of {} "
+                    "please run `python Tests/scripts/update_id_set.py`".format(file_path))
+
+    return is_found
+
+
+def script_valid_in_id_set(file_path, script_set):
```

Comment:
code duplication: pass the first line function as argument (or the result of that)